dom/media/VideoOutput.h

```diff
@@ -118,8 +118,10 @@ class VideoOutput : public DirectMediaTrackListener {
           lastPrincipalHandle, images.LastElement().mFrameID);
     }
 
+    recordreplay::RecordReplayAssert("[RUN-1253] VideoOutput::SendFrames #1");
```

Comment:
Would it not make sense to move this closer to branches inside of `SetCurrentFrames`?

Reply:
Possibly - could go either way.  Mostly I wanted to bookend the call to `SetCurrentFrames` in the caller.  Most of the important logic lives inside `SetCurrentFramesLocked` so added a couple of asserts there.